include/atomic_queue/atomic_queue.h

```diff
@@ -495,13 +496,17 @@ class AtomicQueueB : public AtomicQueueCommon<AtomicQueueB<T, A, NIL, MAXIMIZE_T
     friend void swap(AtomicQueueB& a, AtomicQueueB& b) noexcept {
         a.swap(b);
     }
+
+    allocator_type get_allocator() const noexcept {
+        return static_cast<allocator_type>(static_cast<const AllocatorElements&>(*this));
```

Comment:
The `static_cast` shouldn't be necessary, according to https://en.cppreference.com/w/cpp/named_req/Allocator